lib/shoulda/matchers/active_record/have_db_index_matcher.rb

```diff
@@ -65,6 +65,56 @@ module ActiveRecord
       #
       # @return [HaveDbIndexMatcher]
       #
+      # has an index and an array of columns
+      #
+      #     class CreateBlogs < ActiveRecord::Migration
+      #       def change
+      #         create_table :blogs do |t|
+      #           t.integer :user_id
+      #           t.string :name
+      #         end
+      #
+      #         add_index :blogs, :user_id, :name
+      #       end
+      #     end
+      #
+      #     # RSpec
+      #     RSpec.describe Blog, type: :model do
+      #       it { should have_db_index([:user_id, :name]) }
+      #     end
+      #
+      #     # Minitest (Shoulda)
+      #     class BlogTest < ActiveSupport::TestCase
+      #       should have_db_index([:user_id, :name])
+      #     end
+      #
+      # #### Qualifiers
```

Comment:
Let's remove the additions from this line and lower, it's probably unnecessary to include the `unique` qualifier again, plus it's just a repetitive section.